src/AutoRest.CSharp/Common/Output/Models/Shared/Parameter.cs

```diff
@@ -75,11 +79,16 @@ public static Parameter FromInputParameter(in InputParameter operationParameter,
                 RequestLocation: requestLocation);
         }
 
-        public static string CreateDescription(InputParameter operationParameter, CSharpType type, IEnumerable<string>? values)
+        public static string CreateDescription(InputParameter operationParameter, CSharpType type, IEnumerable<string>? values, Constant? defaultValue = null)
         {
             string description = string.IsNullOrWhiteSpace(operationParameter.Description)
                 ? $"The {operationParameter.Type.Name} to use."
                 : BuilderHelpers.EscapeXmlDocDescription(operationParameter.Description);
+            if (defaultValue != null)
+            {
+                var defaultValueString = defaultValue?.Value is string s ? $"\"{s}\"" : $"{defaultValue?.Value}";
+                description = $"{description}{(description.EndsWith(".") ? "" : ".")} The default is {defaultValueString}";
```

Comment:
This line of information reads a bit weird.
How about `The default value is {}.` or `Defaults to {}.`

Reply:
change to `The default value is {}`